src/main/java/com/mparticle/kits/AdobeKit.kt

```diff
@@ -16,7 +16,8 @@ open class AdobeKit: AdobeKitBase(), KitIntegration.EventListener {
 
     internal val LAUNCH_APP_ID: String = "launchAppId"
 
-    protected var mediaTracker: MediaTracker? = null
+    protected var defaultMediaTracker: MediaTracker? = null
+    protected var mediaTrackers: MutableMap<String, MediaTracker?> = mutableMapOf()
```

Comment:
I think you want this to be:

```
protected var mediaTrackers: MutableMap<String, MediaTracker> = mutableMapOf()
```

I don't think you want to maintain any null trackers. More the session has been created or not.